manifest_staging/charts/secrets-store-csi-driver/templates/csidriver.yaml

```diff
@@ -10,8 +10,8 @@ spec:
   # Added in Kubernetes 1.16 with default mode of Persistent. Secrets store csi driver needs Ephermeral to be set.
   volumeLifecycleModes: 
   - Ephemeral
+  requiresRepublish: {{ .Values.enableSecretRotation }}
   {{- if and (semverCompare ">=1.20-0" .Capabilities.KubeVersion.Version) .Values.tokenRequests }}
```

Comment:
This should be removed if we're moving `requiresRepublish` out of the if check. Let's remove it.